stablehlo/dialect/StablehloAttrs.td

```diff
@@ -20,7 +20,10 @@ limitations under the License.
 include "mlir/IR/OpBase.td"
 include "mlir/IR/TensorEncoding.td"
 
-def StableHLO_Dim : ArrayRefParameter<"int64_t", "Dimension">;
+def StableHLO_Dim : ArrayRefParameter<"int64_t", "Dimension"> {
+  let parser = "mlir::stablehlo::parseIntArray($_parser)";
+  let printer = "mlir::stablehlo::printIntArray($_printer, $_self)";
+}
```

Comment:
Can you also migrate `OutputOperandAlias` to use the same assembly format that's used in MHLO?

Reply:
+1, was thinking the same